exercises/roman-numerals/roman_numerals_test.go

```diff
@@ -21,10 +21,11 @@ func TestRomanNumerals(t *testing.T) {
 			continue
 		}
 		if err != nil && !test.hasError {
+			var _ error = err
 			t.Errorf("ToRomanNumeral(%d) should not return an error.", test.arabic)
 			continue
 		}
-		if actual != test.roman {
+		if !test.hasError && actual != test.roman {
```

Comment:
If anyone asks "Do we still need this? The above cases have `continue`!"

It's still needed because if `err == nil && test.hasError` we will get here and check `actual` when we don't want to.